charts/metallb/templates/speaker.yaml

```diff
@@ -118,6 +270,71 @@ spec:
             - ALL
             add:
             - NET_RAW
+        {{- if .Values.frr.enabled }}
+        volumeMounts:
+          - name: reloader
+            mountPath: /etc/frr_reloader
+         {{- end }}
+      {{- if .Values.frr.enabled }}
+      - name: frr
+        securityContext:
+          capabilities:
+            add: ["NET_ADMIN", "NET_RAW", "SYS_ADMIN", "NET_BIND_SERVICE"]
```

Comment:
Samll formatting nitpick, this should probably be a mutli-line list to match the other `capabilities.add` fields elsewhere in the chart:
```
capabilities:
  add:
  - NET_ADMIN
  - NET_RAW
  - SYS_ADMIN
  - NET_BIND_SERVICE
```

Also, is `SYS_ADMIN` truly required for FRR to function properly?

Reply:
i will fix the list.
@fedepaol or @oribon do you know about `SYS_ADMIN` for FRR?

Reply:
I got this error when I initially started working on adding the frr container:

```
privs_init: initial cap_set_proc failed: Operation not permitted
Wanted caps: cap_net_admin,cap_net_raw,cap_sys_admin=p
Have   caps: cap_chown,cap_dac_override,cap_fowner,cap_fsetid,cap_kill,cap_setgid,cap_setuid,cap_setpcap,cap_net_bind_service,cap_net_raw,cap_sys_chroot,cap_mknod,cap_audit_write,cap_setfcap=ip
starting staticd since zebra is running2021/08/04 09:51:56 STATIC: [EC 100663303] Could not lock pid_file /var/run/frr/staticd.pid (Resource temporarily unavailable), exiting.  Please ensure that the daemon is not already running
privs_init: initial cap_set_proc failed: Operation not permitted
Wanted caps: cap_net_bind_service,cap_net_admin,cap_net_raw,cap_sys_admin=p
Have   caps: cap_chown,cap_dac_override,cap_fowner,cap_fsetid,cap_kill,cap_setgid,cap_setuid,cap_setpcap,cap_net_bind_service,cap_net_raw,cap_sys_chroot,cap_mknod,cap_audit_write,cap_setfcap=ip
```